### DISCUSSION TITLE
NUXT3 npm run build stuck on macos and linux

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [nuxt/nuxt#16809] [Questions]
When I run `npm run build` it hang on You can preview this build using nuxi preview.
I try delete and install node_modules, but it is the same.
<img width="465" alt="Screen Shot 2022-09-17 at 13 56 09" src="https://user-images.githubusercontent.com/5946791/190844813-cf07ff27-6092-47a4-9641-b62f98a71b09.png">

**Comment by sevillaarvin** [comment]
Can you provide a reproduction repo?

**Comment by portwatcher** [comment]
~~I also encouter this issue and my app will never be deployed~~

---

I fixed the issue.
My case is I scheduled a periodical async task in a nitro plugin script's global scope, I moved the scheduling into a function call and it worked.
